lib/inquisition/outputter.rb

```diff
@@ -12,6 +13,13 @@ def self.collection
         @collection ||= {}
       end
 
+      OUTPUTTER_TYPES = {
```

Comment:
It would be good to follow the [consistent structure in our class definition](https://github.com/rubocop-hq/ruby-style-guide#consistent-classes)